data/chains/hyper-evm.json

```diff
@@ -0,0 +1,23 @@
+{
+  "alias": "hyper-evm",
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://hyperliquid.cloud.blockscout.com/api"
+    },
+    "browserUrl": "https://hyperliquid.cloud.blockscout.com/"
+  },
+  "id": "999",
+  "name": "HyperEVM",
+  "providers": [
+    {
+      "alias": "default",
+      "rpcUrl": "https://rpc.hyperliquid.xyz/evm"
+    }
```

Comment:
I recommend adding `https://rpc.hypurrscan.io` as well. A `nslookup` check confirms that it originates from a different source.